main.py

```diff
@@ -111,11 +111,8 @@ async def on_ready():
     global bot
 
     # Make sure this function does not create the officer manager twice
-    if bot.officer_manager is not None:
-        return
-
-    if bot.sql is not None:
-        return
+    if bot.officer_manager is not None or bot.sql is not None:
```

Comment:
As we discussed if bot.sql is false than bot.officer_manager must also be false, since one is defined after the other, so it's enough to just check bot.sql